templates/solutions/infrastructure/form-data.json

```diff
@@ -0,0 +1,44 @@
+{
+  "form": {
+    "/solutions/infrastructure": {
+      "templatePath": "/solutions/infrastructure/index.html",
+      "childrenPaths": [
+        "/solutions/infrastructure/sovereign-cloud",
+        "/solutions/infrastructure/private-cloud-pricing"
+      ],
+      "isModal": true,
+      "modalId": "solutions-infrastructure-modal",
+      "formData": {
+        "title": "Contact Canonical",
+        "formId": "5771",
+        "returnUrl": "/solutions/infrastructure#success",
```

Comment:
```suggestion
        "returnUrl": "/solutions/infrastructure#contact-form-success",
```

Reply:
It seems the [two are identical](https://github.com/canonical/canonical.com/blob/main/templates/base_index.html#L125). Maybe we should consider deprecating one of them